ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -144,33 +150,38 @@ def prepare_lookahead_waypoints(self):
             end_pos = closest_wp_pos + LOOKAHEAD_WPS - 1  # to build waypoint list with a fixed size
             next_waypoints = list(islice(seq, closest_wp_pos, end_pos))  # list of lookahead waypoints
 
-            max_vel = self.mph_to_mps(50)
             if self.traffic is None or self.traffic == -1:
                 self.is_braking = False
-                # no red light
-                target_vel = self.current_velocity
+                # no red light : lets accelerate
+                vel = self.current_velocity
+
                 for i in range(len(next_waypoints) - 1):
-                    target_vel += ACCELERATION
-                    target_vel = min(max_vel, target_vel)
-                    target_vel = max(0.0, target_vel)
-                    self.set_waypoint_velocity(next_waypoints, i, target_vel)
+                    if i == 0:
+                        dist = self.distance(self.current_pose, next_waypoints[0])
+                    else:
+                        dist = self.distance(next_waypoints[i-1].pose.pose, next_waypoints[i])
+
+                    vel += self.accel_limit * dist
```

Comment:
We introduced new parameter to accelerate velocity but still in the launch file (dbw_sim.launch) where we initialize the value, we set this parameter to 1.0. The car will accelerate at the same speed as the multiplier here is 1. Should we increase it a bit?